[2025-01-29T22:47:26.370+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-29T22:47:26.377+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_etl_pipeline.transform_weather_data scheduled__2025-01-28T00:00:00+00:00 [queued]>
[2025-01-29T22:47:26.382+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_etl_pipeline.transform_weather_data scheduled__2025-01-28T00:00:00+00:00 [queued]>
[2025-01-29T22:47:26.382+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-29T22:47:26.389+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): transform_weather_data> on 2025-01-28 00:00:00+00:00
[2025-01-29T22:47:26.392+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=144) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-29T22:47:26.394+0000] {standard_task_runner.py:72} INFO - Started process 147 to run task
[2025-01-29T22:47:26.395+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_etl_pipeline', 'transform_weather_data', 'scheduled__2025-01-28T00:00:00+00:00', '--job-id', '398', '--raw', '--subdir', 'DAGS_FOLDER/app.py', '--cfg-path', '/tmp/tmp7p2w551y']
[2025-01-29T22:47:26.396+0000] {standard_task_runner.py:105} INFO - Job 398: Subtask transform_weather_data
[2025-01-29T22:47:26.419+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_etl_pipeline.transform_weather_data scheduled__2025-01-28T00:00:00+00:00 [running]> on host 10ad9b47e333
[2025-01-29T22:47:26.450+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_etl_pipeline' AIRFLOW_CTX_TASK_ID='transform_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-28T00:00:00+00:00'
[2025-01-29T22:47:26.451+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-29T22:47:26.451+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-29T22:47:26.452+0000] {logging_mixin.py:190} INFO - Current task name:transform_weather_data state:running start_date:2025-01-29 22:47:26.377594+00:00
[2025-01-29T22:47:26.452+0000] {logging_mixin.py:190} INFO - Dag name:weather_etl_pipeline and current dag run status:running
[2025-01-29T22:47:26.452+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-29T22:47:26.452+0000] {python.py:240} INFO - Done. Returned value was: {'latitude': '51.5074', 'longitude': '-0.1278', 'temperature': -2.09, 'windspeed': 49.77, 'winddirection': 104, 'weathercode': 3}
[2025-01-29T22:47:26.463+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-29T22:47:26.463+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_etl_pipeline, task_id=transform_weather_data, run_id=scheduled__2025-01-28T00:00:00+00:00, execution_date=20250128T000000, start_date=20250129T224726, end_date=20250129T224726
[2025-01-29T22:47:26.466+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-29T22:47:26.467+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-29T22:47:26.467+0000] {logging_mixin.py:190} INFO - Dag name:weather_etl_pipeline queued_at:2025-01-29 22:47:21.641957+00:00
[2025-01-29T22:47:26.467+0000] {logging_mixin.py:190} INFO - Task hostname:10ad9b47e333 operator:_PythonDecoratedOperator
[2025-01-29T22:47:26.492+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-29T22:47:26.503+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-29T22:47:26.504+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
